core/core-model/src/main/java/com/hbc/pms/core/model/SensorConfiguration.java

```diff
@@ -4,7 +4,7 @@
 
 @Data
 @Builder
-@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor()

```

Comment:
Why do you change the model's accessibility?

Reply:
Sorry, this was an old test code but I've forgotten to revert